zOSMF-Samples/workflow_sample_wizards_upgrade.xml

```diff
@@ -0,0 +1,327 @@
+ 
+<!-- This step uses a file wizard to create a file (for example, a configuration file), with
+	 variable substitution.                                                             -->
+<step name="create_config">
+        <title>Create a configuration file using a file wizard</title>
+        <description>In this step, you create a configuration file. This step involves using a file wizard.</description>
+        <variableValue name="setting1" required="true"/>
+        <variableValue name="setting2" required="true"/>
+        <variableValue name="setting3" required="false"/>
+		<instructions substitution="false">In this example step, you create an XML configuration file, based on the values that were entered previously.</instructions>
+        <weight>10</weight>
+        <skills>System Programmer</skills>
+        <template> 
+       		<fileTemplate substitution="true">
+       			workflow_sample_file_template0.xml
+       		</fileTemplate>
+ 
+       		<!-- This example specifies <saveAsUnixFile>, but not <saveAsDataset>, because MyProduct requires  
+       			 the configuration file to reside in the UNIX file system. The user is not given an option to 
+				 save to a data set.  This save-as behavior is specific to a file-generation wizard 
+				 (one where <submitAs> is not specified). 
+				 -->
+       		<saveAsUnixFile>/var/MyProduct/config.xml</saveAsUnixFile>
+       	</template>
+</step>
+ 
+<preserveOptions>
+	<!-- The type is set to "refresh" in this version section to indicate not cancelling version 1.0 after upgrade to 2.0 successfully. -->
+	<version value="1.0" type="refresh">
+		<!-- 
+		In the variable mapping section:
+			o - The value of variable setting1 in the calling workflow will be mapped and copied to the variable setting2 in the called workflow when the upgrade happen.
+			o - The variable of value setting2 will not be mapped and copied when the upgrade happen.
+			o - All other variable values in the calling workflow will be mapped and copied with the same name to the called workflow when the upgrade happen.
+		 -->
+		<variableSet defaultChecked="true">
+			<include regExp="\w*"/>
+			<include name="setting1" mapTo="setting2"/>
+			<exclude name ="setting2"/>
+		</variableSet>
+		<!-- 
+			In the step mapping section, all steps except "create_config" in the calling workflow 
+			will be mapped and copied to the steps which have the same name in the called workflow.
+			These data will be copied to the upgraded workflow instance, that is step owner, step 
+			state, step assignee, step notes, job info.
+		 -->
+		<stepSet defaultChecked="true">
+			<include regExp="\w*"/>
+			<exclude name ="create_config"/>
+		</stepSet>	
+		<!--
+			In the workflowHistory section, if the defaultChecked is true, the UI will display the 
+			workflow history check box which checked status.
+		 -->	
+		<workflowHistory defaultChecked = "true"/>
+		<!--  
+		   In the workflowNotes section, nothing is provided, so not checked it
```

Comment:
What is this comment trying to say? It looks like defaultChecked is true, but it says "so not checked it"?

Reply:
It's talking about the workflowNotes section, but no such an element is defined in this sample, so remove this comment.